tests/test_booster/test_plugin/test_low_level_zero_plugin.py

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
How about putting lora tests under folder `tests/test_lora`, there I wrote some util functions for checking the correctness of lora parameters updating